tools/tidy

```diff
@@ -78,8 +77,7 @@ cleanup
 
 find-files | xargs perltidy --pro=.../.perltidyrc
 
-find . -name "*.tdy" | while read -r file
-do
+find-tdy | while read -r file; do
```

Comment:
Can we have a [HEALTHCHECK](https://docs.docker.com/engine/reference/builder/#healthcheck) for this? Or alternatively some way to ensure this continues to work?

Reply:
This is not about running the tidy script *within* the container; it is about running tidy normally after invoking docker-compose (as regular user).